.github/workflows/ci.yml

```diff
@@ -40,11 +40,27 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
 
+      - name: Set branch name
+        shell: bash
+        run: |
+          echo "::set-output name=BRANCH_NAME::${GITHUB_REF##*/}"
+        id: set_branch
+
+      - name: Set asset base URL
+        shell: bash
+        run: |
+          # This corresponds with the method that Oxygen uses to compute the
+          # deployment ID. Changing this will result in incorrect asset URLs.
+          export OXYGEN_STOREFRONT_HASH=`echo -n "${{ secrets.OXYGEN_SHOP_ID }}:${{ secrets.OXYGEN_STOREFRONT_ID }}" | shasum -a 256 | awk '{ print $1 }'`
+          export OXYGEN_DEPLOYMENT_ID=`echo -n "${OXYGEN_STOREFRONT_HASH}:${{ steps.set_branch.outputs.BRANCH_NAME }}:${{ github.event.head_commit.id }}" | shasum -a 256 | awk '{ print substr($1,1,12) }'`
+          echo "::set-output name=BASE_URL::https://cdn.shopify.com/oxygen/${{ secrets.OXYGEN_SHOP_ID }}/${{ secrets.OXYGEN_STOREFRONT_ID }}/${OXYGEN_DEPLOYMENT_ID}/"
```

Comment:
This means we would need to change the Oxygen workflow to fetch the deployment id right?

Reply:
DMS changes will make deployment IDs use the same algorithm as this script, so we won't need to fetch any ID.

Reply:
OMG I missed the hashing in the lines above. 🤦‍♀️